Sources/NIOWebSocket/WebSocketFrame.swift

```diff
@@ -406,3 +406,29 @@ extension WebSocketFrame: CustomDebugStringConvertible {
         "(\(self.description))"
     }
 }
+
+extension WebSocketFrame {
+    /// WebSocketFrame reserved bits option set
+    public struct ReservedBits: OptionSet, Sendable {
+        public var rawValue: UInt8
+
+        public init(rawValue: UInt8) {
```

Comment:
Let's make this `@inlinable` and all the `static var`s as well. That unlocks the meaningful performance improvement.

Reply:
I just noticed the get/set accessors on `firstByte` are not `@inlinable`. Is it worthwhile tagging all of those?

Reply:
I tagged them anyway